tests/acceptance/features/cliCommands/authAppToken.feature

```diff
@@ -0,0 +1,18 @@
+@env-config
+Feature: create auth token
+  As an admin
+  I want to create App Tokens
+  So that I can use 3rd party apps
+
+  Background:
+    Given the following configs have been set:
+      | config                | value    |
+      | OCIS_ADD_RUN_SERVICES | auth-app |
+      | PROXY_ENABLE_APP_AUTH | true     |
+    And user "Alice" has been created with default attributes
+
+
+  Scenario: creates app token via CLI
+    When the administrator creates app token for user "Alice" with expiration time "72h" using the auth-app CLI
+    Then the command should be successful
+    And the command output should be "App token created for Alice\r\n\stoken: %short_app_token_pattern%"
```

Comment:
why we need two different step 	
 * @Then /^the command output (should|should not) contain "([^"]*)"$/

can't we use current strict match in all place

Reply:
@nirajacharya2 I think you need to do that in this PR because your PR adds strict check step. So rather than adding new step, please refactor existing step to work for all scenario

Reply:
>can't we use current strict match in all place

we cannot use stritct match in all scenarios. but we can use contains in all scenarios. no need to have new step.